Changelog.md

```diff
@@ -14,6 +14,10 @@ The way to configure parser is changed. Now all configuration is contained in th
 `Config` struct and can be applied at once. When `serde-types` feature is enabled,
 configuration is serializable.
 
+The way how positions of errors is reported is changed. Now use `error_position()`
```

Comment:
The method of reporting positions of errors has changed - use `error_position()`